server/odc-core/src/main/java/com/oceanbase/odc/core/shared/constant/AuditEventAction.java

```diff
@@ -303,10 +303,9 @@ public enum AuditEventAction implements Translatable {
 
     UPDATE_DATASOURCE,
 
-    CREATE_PROJECT
+    CREATE_PROJECT,
 
-
-    ;
+    CHANGE_SQL_SECURITY_RULE;
```

Comment:
use `UPDATE_` prefix , follow current style.

and, should we distinguish SQL Check Rule & SQL Console Rule ?

Reply:
both sql check rule and sql console rule use the same api to update and the auditing is based on API level interception, so that we can not distinguish SQL Check Rule & SQL Console Rule